diff --git a/modules/defaultServices.jsm b/modules/defaultServices.jsm
index 63c68bf..b91b95b 100644
--- a/modules/defaultServices.jsm
+++ b/modules/defaultServices.jsm
@@ -47,6 +47,15 @@ function frecencyForUrl(host)
 // be maintained in the case the user does not have any logins or frecency.
 // the mediator will enable via login and frecency, and sort by frecency.
 var builtins = [
+
+  {
+    name: "Test",
+    iconURL: "resource://socialdev/testprovider/testprovider/icon.png",
+    workerURL: "resource://socialdev/testprovider/testprovider/worker.js",
+    sidebarURL: "resource://socialdev/testprovider/testprovider/sidebar.htm",
+    URLPrefix: "resource://socialdev/testprovider/testprovider/"
+  }
+
 ];
 
 function installBuiltins() {
